department-of-veterans-affairs/caseflow
# Intake | Add appeal docket selection

Add the options to select the docket for reentries as appeals. Completes
all AC in the ticket.

## Testing instructions
- Start a RAMP Refiling, and select appeal. See all the new shiny options and select one and save. See that the option was saved in the DB.

connects #4139
